src/create.cpp

```diff
@@ -76,7 +76,12 @@ auto create( std::string_view root_, std::string_view indexLocation, const std::
 			continue;
 
 		// open file
+#ifndef _WIN32
 		std::FILE* file{ std::fopen( path.c_str(), "rb" ) };
+#else
+		std::FILE* file = nullptr;
```

Comment:
please use same initialization style as above

Reply:
it compiles to the same thing, i get its a style thing though